common/motion_utils/include/motion_utils/trajectory/path_with_lane_id.hpp

```diff
@@ -99,6 +99,11 @@ inline size_t findNearestSegmentIndexFromLaneId(
 
   return nearest_idx;
 }
+
+// NOTE: rear_to_cog is supposed to be positive
```

Comment:
This function is hard to understand for users. Please write a detailed description.

```suggestion
// @brief Calculates the path to be followed by the rear wheel center in order to make the vehicle center follow the input path
// @param [in] path with position to be followed by the center of the vehicle
// @param [out] PathWithLaneId to be followed by the rear wheel center follow to make the vehicle center follow the input path
// NOTE: rear_to_cog is supposed to be positive
```